.github/workflows/ci.yml

```
@@ -0,0 +1,188 @@
+      - name: Generate docs
+        run: |
+          cd go
+          mkdir -p ../docs
+          go doc -all ./... > ../docs/api.md
+          cp README.md ../docs/go.md
+          cd ..
+          cp sh/README.md docs/sh.md
+          cp README.md docs/index.md
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.4)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)